macros/multiple_databases/to_boolean.sql

```diff
@@ -14,4 +14,40 @@
     {%- endif -%}
 {%- endif -%}
 
+{# Warning if type casting will introduce null values for at least 1 record. #}
+{% if table is defined %}
```

Comment:
Make the argument name consistent over the macros. Does, as we discussed, _relation_ work?